src/err/mod.rs

```diff
@@ -431,13 +431,33 @@ impl PyErr {
     }
 
     /// Prints a standard traceback to `sys.stderr`.
+    #[cfg(Py_3_12)]
+    pub fn display(&self, py: Python<'_>) {
+        unsafe { ffi::PyErr_DisplayException(self.value(py).as_ptr()) }
+    }
+
+    /// Prints a standard traceback to `sys.stderr`.
+    #[cfg(not(Py_3_12))]
+    pub fn display(&self, py: Python<'_>) {
+        unsafe {
+            ffi::PyErr_Display(
+                self.get_type(py).as_ptr(),
+                self.value(py).as_ptr(),
+                self.traceback(py)
+                    .map_or(std::ptr::null_mut(), PyTraceback::as_ptr),
+            )
+        }
+    }
```

Comment:
```suggestion
    pub fn display(&self, py: Python<'_>) {
        unsafe { 
            if cfg!(Py_3_12) {
                ffi::PyErr_DisplayException(self.value(py).as_ptr())
            } else {
               ffi::PyErr_Display(
                  self.get_type(py).as_ptr(),
                  self.value(py).as_ptr(),
                  self.traceback(py)
                      .map_or(std::ptr::null_mut(), PyTraceback::as_ptr),
            )
        }
    }
```
Otherwise this function will show up as being unavailable on Python 3.12

Reply:
Good catch, thanks - will fixup when I get a moment